pom.xml

```diff
@@ -138,7 +138,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>mod-pubsub-client</artifactId>
-      <version>1.2.3</version>
+      <version>1.3.0-SNAPSHOT</version>
```

Comment:
That's not part of normal openjdk 11 migration.

Reply:
Yes, without pub-sub-client upgraded we have a runtime issue because the client can not get module version using pom.